llvm/test/CodeGen/AMDGPU/skip-optimize-agpr-phi-without-subreg-use.mir

```diff
@@ -0,0 +1,64 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx908 -verify-machineinstrs -run-pass si-fold-operands -o - %s | FileCheck %s
+
```

Comment:
Can you just merge this with test/CodeGen/AMDGPU/fold-agpr-phis.mir instead of a new test file?